### PR TITLE
Apply changes for core 4.3.0

### PR DESCRIPTION
**Pull request by craigpalermo**
## 4.3.0 (tentative)
Mon, 04 Jan 2021 16:21:42 GMT

### Minor changes

- Add Shimmer component
- Add Accessibility SkipMenu component
- Add helperText prop for showing error messages in forms
- add a new component GenericBanner to core

### Patches

- Fix bug when using RatingBadge with color but not bg prop
- Resolve warnings in Storybook and remove use of deprecated styled().attrs({}) syntax
- Update Text Extension Display Names
- Change withComponent to as

### CODE REVIEW

File: packages/autocomplete/package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "@rushstack/eslint-config": "^2.3.1"
   },
   "peerDependencies": {
-    "pcln-design-system": "^4.2.1",
+    "pcln-design-system": "4.3.0",
```
Comment (sdalonzo) [inline packages/autocomplete/package.json:58]: peers should never be this specific

File: packages/menu/package.json [hunk 2]
```diff
@@ -48,9 +48,9 @@
     "@rushstack/eslint-config": "^2.3.1"
   },
   "peerDependencies": {
-    "pcln-design-system": "^4.2.1",
-    "pcln-icons": "^4.0.0",
-    "pcln-popover": "^4.0.1",
+    "pcln-design-system": "4.3.0",
```
Comment (sdalonzo) [inline packages/menu/package.json:51]: same here.

File: packages/modal/package.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
     "@rushstack/eslint-config": "^2.3.1"
   },
   "peerDependencies": {
-    "pcln-design-system": "^4.2.1",
+    "pcln-design-system": "4.3.0",
```
Comment (sdalonzo) [inline packages/modal/package.json:54]: and here

File: packages/popover/package.json [hunk 4]
```diff
@@ -57,7 +57,7 @@
     "@rushstack/eslint-config": "^2.3.1"
   },
   "peerDependencies": {
-    "pcln-design-system": "^4.2.1",
+    "pcln-design-system": "4.3.0",
```
Comment (sdalonzo) [inline packages/popover/package.json:60]: same

File: packages/slider/package.json [hunk 5]
```diff
@@ -63,7 +63,7 @@
     "@rushstack/eslint-config": "^2.3.1"
   },
   "peerDependencies": {
-    "pcln-design-system": "^4.2.1",
+    "pcln-design-system": "4.3.0",
```
Comment (sdalonzo) [inline packages/slider/package.json:66]: you get the idea ;)

**Review by sdalonzo — CHANGES_REQUESTED** [review]
Peers need to be careted. is 4.3 actually required?

**Comment by craigpalermo** [comment]
> Peers need to be careted. is 4.3 actually required?

I don't think `4.3.0` is strictly required by any sub packages, but I think it makes sense to keep the peers up-to-date to encourage consumers to keep all of their DS-related packages at the latest instead of mixing versions within the same major because it seems like less work.

**Review by sdalonzo — APPROVED** [review]

File: packages/core/CHANGELOG.json [hunk 6]
```diff
@@ -1,6 +1,57 @@
...
+        ],
+        "patch": [
+          {
+            "comment": "Fix bug when using RatingBadge with color but not bg prop",
+            "author": "Craig Palermo <3260642+craigpalermo@users.noreply.github.com>",
+            "commit": "366e6521107e6ccf61b2c59cfed536067a6eed55"
+          },
+          {
+            "comment": "Resolve warnings in Storybook and remove use of deprecated styled().attrs({}) syntax",
+            "author": "Craig Palermo <3260642+craigpalermo@users.noreply.github.com>",
```
Comment (pcln-james) [inline packages/core/CHANGELOG.json:39]: weird most of our emails didn't work

**Review by pcln-james — APPROVED** [review]
LGTM but we should look into the email issue

**Comment by craigpalermo** [comment]
> LGTM but we should look into the email issue

Oh yeah hmm, agreed. We'll have to look into that.

**Pull request merged**